languoids/tree/choc1280/embe1258/sanj1278/uppe1440/embe1261/md.ini

```diff
@@ -31,7 +31,8 @@ lexvo =
 	Emberá-Tadó [en]
 hhbib_lgcode = 
 	Embera-Tado
-	Emberá-Tadó 
+	Emberá-Tadó
+	Emberá-Tadó
```

Comment:
Something seems to be going on here (and in two more files) with the doubling of names. Any idea what could have caused this @d97hah ?